client/src/components/CustomFields.js

```diff
@@ -511,6 +691,14 @@ const CustomField = ({
     >
       {helpText && <HelpBlock>{helpText}</HelpBlock>}
```

Comment:
when `helpText` is defined, ArrayOfAnetObjectsField doesn't seem to render the `HelpBlock`

Reply:
It looks like many other custom field components also do not render their children, so the help text would be missing in other places as well. I just copied behaviour from a similar custom field component.

Reply:
The ArrayOfObjectsField that I took as an example also doesn't show the helpText. Fixed all three now (ArrayOfObjectsField and AnetObjectField, ArrayOfAnetObjectFields).

Reply:
:+1: